### PR TITLE
Add a raw not where clause to the query

### PR DESCRIPTION
**Pull request by hachi-zzq**
```php
    /**
     * Add a raw not where clause to the query
     *
     * @param string $sql
     * @param mixed $bindings
     * @return \Illuminate\Database\Query\Builder|static
     */
    public function notWhereRaw($sql, $bindings = [])
    {
        return $this->whereRaw("Not ($sql)", $bindings, 'and');
    }
```

### CODE REVIEW

File: src/Illuminate/Database/Query/Builder.php [hunk 1]
```diff
@@ -756,6 +756,18 @@ public function orWhereRaw($sql, $bindings = [])
...
     }
 
+    /**
+     * Add a raw not where clause to the query.
+     *
+     * @param string $sql
+     * @param mixed $bindings
+     * @return \Illuminate\Database\Query\Builder|static
+     */
+    public function notWhereRaw($sql, $bindings = [])
```
Comment (jmarcher) [inline src/Illuminate/Database/Query/Builder.php]: The method should be named whereNotRaw

**Comment by jmarcher** [comment]
Can you provide some tests?

**Comment by cyrildewit** [comment]
According to Laravel's [Contribution Guide](https://laravel.com/docs/5.6/contributions) on [laravel.com](https://laravel.com/docs/5), the `@param` attribute should be followed by two spaces, the argument type, two more spaces, and finally the variable name.

```php
/**
 * Add a raw not where clause to the query.
 *
 * @param  string  $sql
 * @param  mixed  $bindings
 * @return \Illuminate\Database\Query\Builder|static
*/
```

Skip this if it's done automatically.

**Comment by taylorotwell** [comment]
Couldn't you just use whereRaw and add the not yourself?

**Pull request closed without merging**